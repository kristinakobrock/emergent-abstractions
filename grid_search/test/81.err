  0%|                                                                                                     | 0/50 [00:00<?, ?it/s]  4%|###7                                                                                         | 2/50 [00:00<00:02, 19.58it/s] 10%|#########3                                                                                   | 5/50 [00:00<00:01, 25.00it/s] 16%|##############8                                                                              | 8/50 [00:00<00:01, 24.76it/s] 22%|####################2                                                                       | 11/50 [00:00<00:01, 24.72it/s] 28%|#########################7                                                                  | 14/50 [00:00<00:01, 21.91it/s] 34%|###############################2                                                            | 17/50 [00:00<00:01, 22.19it/s] 40%|####################################8                                                       | 20/50 [00:00<00:01, 20.81it/s] 46%|##########################################3                                                 | 23/50 [00:01<00:01, 21.75it/s] 52%|###############################################8                                            | 26/50 [00:01<00:01, 23.52it/s] 58%|#####################################################3                                      | 29/50 [00:01<00:00, 23.86it/s] 64%|##########################################################8                                 | 32/50 [00:01<00:00, 24.14it/s] 70%|################################################################3                           | 35/50 [00:01<00:00, 25.17it/s] 78%|#######################################################################7                    | 39/50 [00:01<00:00, 28.85it/s] 84%|#############################################################################2              | 42/50 [00:01<00:00, 28.85it/s] 90%|##################################################################################8         | 45/50 [00:01<00:00, 28.59it/s] 96%|########################################################################################3   | 48/50 [00:01<00:00, 28.34it/s]100%|############################################################################################| 50/50 [00:01<00:00, 25.44it/s]
  0%|                                                                                                     | 0/13 [00:00<?, ?it/s] 15%|##############3                                                                              | 2/13 [00:00<00:00, 18.63it/s] 31%|############################6                                                                | 4/13 [00:00<00:00, 18.99it/s] 54%|##################################################                                           | 7/13 [00:00<00:00, 23.35it/s] 77%|######################################################################7                     | 10/13 [00:00<00:00, 22.83it/s]100%|############################################################################################| 13/13 [00:00<00:00, 21.12it/s]100%|############################################################################################| 13/13 [00:00<00:00, 21.27it/s]
