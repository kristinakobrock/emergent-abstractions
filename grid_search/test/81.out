# ["--attributes=5", "--values=16", "--game_size=10", "--vocab_size_factor=3", "--n_epochs=60", "--batch_size=64", "--learning_rate=0.0005", "--hidden_size=128", "--temp_update=0.99", "--temperature=2"]
Creating train_ds and val_ds...

Creating test_ds...
{"loss": 0.6598395109176636, "acc": 0.631897509098053, "length": 3.5700221061706543, "mode": "train", "epoch": 1}
{"loss": 0.6593602299690247, "acc": 0.6191406846046448, "length": 4.0, "mode": "test", "epoch": 1}
{"loss": 0.6556149125099182, "acc": 0.6334007978439331, "length": 3.5859808921813965, "mode": "train", "epoch": 2}
{"loss": 0.6542028188705444, "acc": 0.6236329078674316, "length": 4.0, "mode": "test", "epoch": 2}
{"loss": 0.65450119972229, "acc": 0.6335687637329102, "length": 3.6406688690185547, "mode": "train", "epoch": 3}
{"loss": 0.6461231112480164, "acc": 0.6341796517372131, "length": 4.0, "mode": "test", "epoch": 3}
{"loss": 0.6513577103614807, "acc": 0.6352642774581909, "length": 3.7455391883850098, "mode": "train", "epoch": 4}
{"loss": 0.6413379311561584, "acc": 0.6365234851837158, "length": 4.0, "mode": "test", "epoch": 4}
{"loss": 0.6423340439796448, "acc": 0.6400266289710999, "length": 3.8144092559814453, "mode": "train", "epoch": 5}
{"loss": 0.6315155029296875, "acc": 0.6429687142372131, "length": 4.0, "mode": "test", "epoch": 5}
{"loss": 0.624054491519928, "acc": 0.6571223735809326, "length": 3.884150981903076, "mode": "train", "epoch": 6}
{"loss": 0.6074793338775635, "acc": 0.6701172590255737, "length": 4.0, "mode": "test", "epoch": 6}
{"loss": 0.6027750968933105, "acc": 0.6748433709144592, "length": 3.924459218978882, "mode": "train", "epoch": 7}
{"loss": 0.6096394658088684, "acc": 0.6693359017372131, "length": 4.0, "mode": "test", "epoch": 7}
{"loss": 0.5904277563095093, "acc": 0.6860567927360535, "length": 3.940782308578491, "mode": "train", "epoch": 8}
{"loss": 0.6143012046813965, "acc": 0.6798827648162842, "length": 4.0, "mode": "test", "epoch": 8}
{"loss": 0.5740526914596558, "acc": 0.7008602023124695, "length": 3.9494526386260986, "mode": "train", "epoch": 9}
{"loss": 0.6034172177314758, "acc": 0.6884765625, "length": 4.0, "mode": "test", "epoch": 9}
{"loss": 0.557573676109314, "acc": 0.7197681069374084, "length": 3.959456443786621, "mode": "train", "epoch": 10}
{"loss": 0.6211786866188049, "acc": 0.6898437142372131, "length": 4.0, "mode": "test", "epoch": 10}
{"loss": 0.547843337059021, "acc": 0.7233178019523621, "length": 3.973872661590576, "mode": "train", "epoch": 11}
{"loss": 0.6377111673355103, "acc": 0.6917967796325684, "length": 4.0, "mode": "test", "epoch": 11}
{"loss": 0.539579451084137, "acc": 0.7287731170654297, "length": 3.981060028076172, "mode": "train", "epoch": 12}
{"loss": 0.6361274123191833, "acc": 0.6966796517372131, "length": 4.0, "mode": "test", "epoch": 12}
{"loss": 0.5325330495834351, "acc": 0.7333385944366455, "length": 3.9842846393585205, "mode": "train", "epoch": 13}
{"loss": 0.6424113512039185, "acc": 0.6904296278953552, "length": 4.0, "mode": "test", "epoch": 13}
{"loss": 0.5263924598693848, "acc": 0.737928032875061, "length": 3.988461494445801, "mode": "train", "epoch": 14}
{"loss": 0.6484262347221375, "acc": 0.6910156607627869, "length": 4.0, "mode": "test", "epoch": 14}
{"loss": 0.5172828435897827, "acc": 0.7407732605934143, "length": 3.990729331970215, "mode": "train", "epoch": 15}
{"loss": 0.6454258561134338, "acc": 0.6951172351837158, "length": 4.0, "mode": "test", "epoch": 15}
{"loss": 0.5154244899749756, "acc": 0.749199390411377, "length": 3.992032051086426, "mode": "train", "epoch": 16}
{"loss": 0.6538682579994202, "acc": 0.6980468034744263, "length": 4.0, "mode": "test", "epoch": 16}
{"loss": 0.5109302401542664, "acc": 0.7512262463569641, "length": 3.992543935775757, "mode": "train", "epoch": 17}
{"loss": 0.6378260850906372, "acc": 0.7041015625, "length": 4.0, "mode": "test", "epoch": 17}
{"loss": 0.5051283836364746, "acc": 0.7554430961608887, "length": 3.9931695461273193, "mode": "train", "epoch": 18}
{"loss": 0.6181285381317139, "acc": 0.7148436903953552, "length": 4.0, "mode": "test", "epoch": 18}
{"loss": 0.4979613721370697, "acc": 0.7583338022232056, "length": 3.9935123920440674, "mode": "train", "epoch": 19}
{"loss": 0.6194649934768677, "acc": 0.7093749642372131, "length": 4.0, "mode": "test", "epoch": 19}
{"loss": 0.49391430616378784, "acc": 0.7621305584907532, "length": 3.994418144226074, "mode": "train", "epoch": 20}
{"loss": 0.5994974970817566, "acc": 0.7230467796325684, "length": 4.0, "mode": "test", "epoch": 20}
{"loss": 0.4887453317642212, "acc": 0.7662737369537354, "length": 3.993495225906372, "mode": "train", "epoch": 21}
{"loss": 0.5892279744148254, "acc": 0.7238281965255737, "length": 4.0, "mode": "test", "epoch": 21}
{"loss": 0.48139292001724243, "acc": 0.7720366716384888, "length": 3.994799852371216, "mode": "train", "epoch": 22}
{"loss": 0.5664948225021362, "acc": 0.732421875, "length": 4.0, "mode": "test", "epoch": 22}
{"loss": 0.4747603237628937, "acc": 0.7749195098876953, "length": 3.995008945465088, "mode": "train", "epoch": 23}
{"loss": 0.5707387924194336, "acc": 0.7318359017372131, "length": 4.0, "mode": "test", "epoch": 23}
{"loss": 0.4706161320209503, "acc": 0.776263415813446, "length": 3.995319128036499, "mode": "train", "epoch": 24}
{"loss": 0.5536044239997864, "acc": 0.7408203482627869, "length": 4.0, "mode": "test", "epoch": 24}
{"loss": 0.4649931490421295, "acc": 0.7810317873954773, "length": 3.9954116344451904, "mode": "train", "epoch": 25}
{"loss": 0.5529965162277222, "acc": 0.73828125, "length": 4.0, "mode": "test", "epoch": 25}
{"loss": 0.45558592677116394, "acc": 0.7825652360916138, "length": 3.9956953525543213, "mode": "train", "epoch": 26}
{"loss": 0.5464012026786804, "acc": 0.7478514909744263, "length": 4.0, "mode": "test", "epoch": 26}
{"loss": 0.45018917322158813, "acc": 0.7878943085670471, "length": 3.9964356422424316, "mode": "train", "epoch": 27}
{"loss": 0.5520727634429932, "acc": 0.7451171875, "length": 4.0, "mode": "test", "epoch": 27}
{"loss": 0.44534361362457275, "acc": 0.7883784174919128, "length": 3.9960131645202637, "mode": "train", "epoch": 28}
{"loss": 0.5404141545295715, "acc": 0.7433593273162842, "length": 4.0, "mode": "test", "epoch": 28}
{"loss": 0.4422149360179901, "acc": 0.794803261756897, "length": 3.9966928958892822, "mode": "train", "epoch": 29}
{"loss": 0.5407609939575195, "acc": 0.7404297590255737, "length": 4.0, "mode": "test", "epoch": 29}
{"loss": 0.4361991584300995, "acc": 0.7963353395462036, "length": 3.9965250492095947, "mode": "train", "epoch": 30}
{"loss": 0.527379035949707, "acc": 0.7542967796325684, "length": 4.0, "mode": "test", "epoch": 30}
{"loss": 0.4334680438041687, "acc": 0.8002409934997559, "length": 3.996242046356201, "mode": "train", "epoch": 31}
{"loss": 0.5282749533653259, "acc": 0.7583984732627869, "length": 4.0, "mode": "test", "epoch": 31}
{"loss": 0.42561882734298706, "acc": 0.8045904636383057, "length": 3.996288299560547, "mode": "train", "epoch": 32}
{"loss": 0.5172656774520874, "acc": 0.7640624642372131, "length": 4.0, "mode": "test", "epoch": 32}
{"loss": 0.4224143624305725, "acc": 0.8069102168083191, "length": 3.9960756301879883, "mode": "train", "epoch": 33}
{"loss": 0.5103830695152283, "acc": 0.7718749642372131, "length": 4.0, "mode": "test", "epoch": 33}
{"loss": 0.419222891330719, "acc": 0.8069602251052856, "length": 3.9953174591064453, "mode": "train", "epoch": 34}
{"loss": 0.5220521688461304, "acc": 0.7626953125, "length": 4.0, "mode": "test", "epoch": 34}
{"loss": 0.4169541597366333, "acc": 0.8113158941268921, "length": 3.9963722229003906, "mode": "train", "epoch": 35}
{"loss": 0.5150371789932251, "acc": 0.7652344107627869, "length": 4.0, "mode": "test", "epoch": 35}
{"loss": 0.4127429723739624, "acc": 0.8132724761962891, "length": 3.995896577835083, "mode": "train", "epoch": 36}
{"loss": 0.5102677941322327, "acc": 0.7640625238418579, "length": 4.0, "mode": "test", "epoch": 36}
{"loss": 0.4081065058708191, "acc": 0.8191996812820435, "length": 3.9958105087280273, "mode": "train", "epoch": 37}
{"loss": 0.500459611415863, "acc": 0.7685546875, "length": 4.0, "mode": "test", "epoch": 37}
{"loss": 0.4050906002521515, "acc": 0.8220193982124329, "length": 3.996005058288574, "mode": "train", "epoch": 38}
{"loss": 0.5162112712860107, "acc": 0.7667969465255737, "length": 4.0, "mode": "test", "epoch": 38}
{"loss": 0.40191876888275146, "acc": 0.8197572827339172, "length": 3.996215581893921, "mode": "train", "epoch": 39}
{"loss": 0.49893006682395935, "acc": 0.7658203840255737, "length": 4.0, "mode": "test", "epoch": 39}
{"loss": 0.3963577151298523, "acc": 0.8253231644630432, "length": 3.9965593814849854, "mode": "train", "epoch": 40}
{"loss": 0.5017589330673218, "acc": 0.7699218392372131, "length": 4.0, "mode": "test", "epoch": 40}
{"loss": 0.3952775001525879, "acc": 0.8256959915161133, "length": 3.996495485305786, "mode": "train", "epoch": 41}
{"loss": 0.5015472769737244, "acc": 0.7707030773162842, "length": 4.0, "mode": "test", "epoch": 41}
{"loss": 0.39160093665122986, "acc": 0.8281433582305908, "length": 3.996931552886963, "mode": "train", "epoch": 42}
{"loss": 0.5048205256462097, "acc": 0.7664062976837158, "length": 4.0, "mode": "test", "epoch": 42}
{"loss": 0.38694557547569275, "acc": 0.830258846282959, "length": 3.9957432746887207, "mode": "train", "epoch": 43}
{"loss": 0.5079089999198914, "acc": 0.7628905773162842, "length": 4.0, "mode": "test", "epoch": 43}
{"loss": 0.3816152811050415, "acc": 0.8317351341247559, "length": 3.995748281478882, "mode": "train", "epoch": 44}
{"loss": 0.4967077970504761, "acc": 0.7800781726837158, "length": 4.0, "mode": "test", "epoch": 44}
{"loss": 0.37998515367507935, "acc": 0.8352966904640198, "length": 3.9964957237243652, "mode": "train", "epoch": 45}
{"loss": 0.49935293197631836, "acc": 0.7781250476837158, "length": 4.0, "mode": "test", "epoch": 45}
{"loss": 0.3782140910625458, "acc": 0.8367161154747009, "length": 3.9973719120025635, "mode": "train", "epoch": 46}
{"loss": 0.4909136891365051, "acc": 0.7830078601837158, "length": 4.0, "mode": "test", "epoch": 46}
{"loss": 0.3746606707572937, "acc": 0.8361798524856567, "length": 3.9979512691497803, "mode": "train", "epoch": 47}
{"loss": 0.47934556007385254, "acc": 0.7853516340255737, "length": 4.0, "mode": "test", "epoch": 47}
{"loss": 0.37048280239105225, "acc": 0.8396352529525757, "length": 3.9967823028564453, "mode": "train", "epoch": 48}
{"loss": 0.4661332964897156, "acc": 0.7923827767372131, "length": 4.0, "mode": "test", "epoch": 48}
{"loss": 0.3626108765602112, "acc": 0.8441253304481506, "length": 3.9973931312561035, "mode": "train", "epoch": 49}
{"loss": 0.47514480352401733, "acc": 0.7947264909744263, "length": 4.0, "mode": "test", "epoch": 49}
{"loss": 0.36393672227859497, "acc": 0.8412030339241028, "length": 3.9979281425476074, "mode": "train", "epoch": 50}
{"loss": 0.46996569633483887, "acc": 0.7951172590255737, "length": 4.0, "mode": "test", "epoch": 50}
{"loss": 0.35943812131881714, "acc": 0.848415732383728, "length": 3.997898578643799, "mode": "train", "epoch": 51}
{"loss": 0.48149874806404114, "acc": 0.7914062142372131, "length": 4.0, "mode": "test", "epoch": 51}
{"loss": 0.35143202543258667, "acc": 0.8496899008750916, "length": 3.997074604034424, "mode": "train", "epoch": 52}
{"loss": 0.4625217318534851, "acc": 0.7992187142372131, "length": 4.0, "mode": "test", "epoch": 52}
{"loss": 0.3494183123111725, "acc": 0.8508807420730591, "length": 3.9965553283691406, "mode": "train", "epoch": 53}
{"loss": 0.4714546799659729, "acc": 0.7980468273162842, "length": 4.0, "mode": "test", "epoch": 53}
{"loss": 0.34853383898735046, "acc": 0.8499560952186584, "length": 3.9962992668151855, "mode": "train", "epoch": 54}
{"loss": 0.45669734477996826, "acc": 0.8031249642372131, "length": 4.0, "mode": "test", "epoch": 54}
{"loss": 0.34215307235717773, "acc": 0.8569433093070984, "length": 3.998731851577759, "mode": "train", "epoch": 55}
{"loss": 0.4430757761001587, "acc": 0.8023437857627869, "length": 4.0, "mode": "test", "epoch": 55}
{"loss": 0.3359747529029846, "acc": 0.8590532541275024, "length": 3.9975359439849854, "mode": "train", "epoch": 56}
{"loss": 0.4530336856842041, "acc": 0.8064452409744263, "length": 4.0, "mode": "test", "epoch": 56}
{"loss": 0.33528435230255127, "acc": 0.8578671216964722, "length": 3.9988110065460205, "mode": "train", "epoch": 57}
{"loss": 0.4361327290534973, "acc": 0.8113280534744263, "length": 4.0, "mode": "test", "epoch": 57}
{"loss": 0.3312075734138489, "acc": 0.8619392514228821, "length": 3.9982707500457764, "mode": "train", "epoch": 58}
{"loss": 0.42975014448165894, "acc": 0.8138671517372131, "length": 4.0, "mode": "test", "epoch": 58}
{"loss": 0.3243313133716583, "acc": 0.8645275235176086, "length": 3.998689889907837, "mode": "train", "epoch": 59}
{"loss": 0.43383294343948364, "acc": 0.8109374642372131, "length": 4.0, "mode": "test", "epoch": 59}
{"loss": 0.32216235995292664, "acc": 0.8639814853668213, "length": 3.9988136291503906, "mode": "train", "epoch": 60}
{"loss": 0.4484965205192566, "acc": 0.8175782561302185, "length": 4.0, "mode": "test", "epoch": 60}
test accuracy: 0.7658572196960449
